src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -12465,4 +12465,52 @@ bool GetHeatPumpWaterHeaterNodeNumber(EnergyPlusData &state, int const NodeNumbe
     return HeatPumpWaterHeaterNodeException;
 }
 
+int GetHeatPumpWaterHeaterAirInletNodeNum(EnergyPlusData &state, int HPNum)
+{
+    int HeatPumpWaterHeaterAirInletNodeNum = 0;
+    if (state.dataWaterThermalTanks->getWaterThermalTankInputFlag) {
+        GetWaterThermalTankInput(state);
+        state.dataWaterThermalTanks->getWaterThermalTankInputFlag = false;
+    }
+
+    if (HPNum > 0 && HPNum <= state.dataWaterThermalTanks->numHeatPumpWaterHeater) {
+        HeatPumpWaterHeaterAirInletNodeNum = state.dataWaterThermalTanks->HPWaterHeater(HPNum).HeatPumpAirInletNode;
+    }
+
+    return HeatPumpWaterHeaterAirInletNodeNum;
+}
+
+int GetHeatPumpWaterHeaterAirOutletNodeNum(EnergyPlusData &state, int HPNum)
+{
+    int HeatPumpWaterHeaterAirOutletNodeNum = 0;
+    if (state.dataWaterThermalTanks->getWaterThermalTankInputFlag) {
+        GetWaterThermalTankInput(state);
+        state.dataWaterThermalTanks->getWaterThermalTankInputFlag = false;
+    }
```

Comment:
Is this real?  We actually need to call GetInput even on functions when we are passing in HPNum??